modules/integration/tests-platform/src/test/java/org/wso2/automation/platform/tests/apim/is/SingleSignOnTestCase.java

```diff
@@ -151,6 +153,7 @@ public void init() throws APIManagerIntegrationTestException {
     @AfterClass(alwaysRun = true)
     public void destroy() throws Exception {
         super.cleanup();
+        deleteApplication();
```

Comment:
move this line before above line, After super.cleanup method couldn't do any of the server requests.